### DISCUSSION TITLE
Subscriptions with GraphQL-SSE

### DISCUSSION THREAD

**Discussion by oliveiracdz** [ChilliCream/hotchocolate#5961] [Q&A] [answered]
[HC 13](https://chillicream.com/blog/2023/02/08/new-in-hot-chocolate-13#graphql-over-internet) already supports subscriptions with SSE, right?
My subscriptions is working well with websockets, but the messages do not arrive when I try "GraphQL SSE" [as subscription protocol](https://chillicream.com/static/0be132fce22234fa993f7136ad22b4d5/a7e6e/bcp-1.png).
There is any additional configuration that need to be made on server?

Thank you!

**Comment by PascalSenn** [comment, marked as answer]
@oliveiracdz  GraphQL SSE should work out of the box with HC 13. Do you not see the subscription events in the BCP or in you application?
Does it also not work when you use the newest version on eat.bananacakepop.com?

**Reply by oliveiracdz** [reply]
It really works, @PascalSenn. Thank you for the confirmation!
The problem was a conflict with my custom ETag middleware.

Thanks! 🙂

**Reply by Arsync** [reply]
What about StrawberryShake, can we enable GraphQL SSE now instead of using websockets? Can't find any example of how to do this.

**Reply by PascalSenn** [reply]
@Arsync  You just specify "HTTP" as the subscription transport and it will work

**Reply by Arsync** [reply]
It stop working after updated to v15, is there something changed in SSE support?

**Reply by MartinIsaksson** [reply]
Hello!

Seems like I have the same issue, using v15 as well. 
When debugging, all the internals works nicely but the active client listening on the subscription never receives the messages. I use both ordinary topics as well as dynamic topics. I use SSE. 
I'm using Azure function with a CosmosDB change feed to trigger the subscription flow.
I've tried the latest version of eat.bananacakepop.com as well with no luck.

Anything else I can try to find the culprit of the issue? 

Thanks in advance.
